libs/extensions/tests_extensions/test_loader.py

```diff
@@ -0,0 +1,646 @@
+'''
+Tests for loader.py
+'''
+import os
+import re
+from unittest import TestCase
+from unittest.mock import patch
+
+from libs.extensions.loader import Extension, generate_dict_with_jsonfield_and_reportfield, load_extensions, \
+    append_extensions_misp_types
+from libs.kafka.logging import LogMessage
+
+
+def get_path(relpath):
+    '''
+    get_path will get the path
+    for a given file.
+    @param relpath will be the relative path to a file
+    @return returns the path to the file.
+    '''
+    abspath = os.path.abspath(__file__)
+    dirname = os.path.dirname(abspath)
+    filepath = os.path.join(dirname, relpath)
+    return filepath
```

Comment:
It would be propably be better to provide this snippet as a common function and pass the __file__ and relpath argument ;)

Reply:
Where would you place it?

Reply:
Thats up to you ;) I think `libs\core` would be a good place?

Reply:
I am not sure, but I think I have to place this function within the file, I want to get access from. In this case the test files, or else, the logic won't work anymore.

Reply:
Should be fixed.